package.json

```diff
@@ -16,7 +16,7 @@
     "build": "jest --config jest.build.config.js --no-cache",
     "postbuild": "ts-node scripts/postBuild.ts",
     "prettier": "prettier --write --config prettier.config.js .",
-    "prettier:check": "prettier --config prettier.config.js --check .",
+    "prettier:check": "prettier --config prettier.config.js --check '**/*.{js,jsx,ts,tsx}'",
```

Comment:
please explain why we need that

Reply:
Due to a bug in Prettier, when running `--check` Prettier checks all the files even non-supported ones.  This caused the build step on Github to fail. Hence my change.

Reply:
Commenting for reference: https://github.com/xFAANG/askql/issues/221